src/lib/containers/EntityIdList.tsx

```diff
@@ -29,12 +29,16 @@ const EntityIdList: React.FC<EntityIdListProps> = props => {
 
     getEntityHeadersByIds(entityIdList, accessToken)
       .then(entity => {
-        const newEntityList = entity.results.map(el => {
-          return (
-            <EntityLink entity={el} key={el.id} displayTextField={'name'} />
-          )
-        })
-        setEntityLinkArray(newEntityList)
+        if (mounted) {
```

Comment:
mounted should be checked in the asynchronous callback, where it is possible for mounted to be false when this is invoked